packages/resend/emails/digest.tsx

```
@@ -0,0 +1,767 @@
+                      key={index}
+                      className="mb-[8px] bg-white rounded-[6px] p-[10px] border-solid border-[1px] border-blue-200"
+                    >
+                      <Text className="text-[14px] font-bold text-gray-800 m-0">
+                        {item.subject}
+                      </Text>
+                      <Text className="text-[12px] text-gray-600 mt-[1px] mb-[10px] leading-[15px]">
+                        {item.from}
+                      </Text>
+                      <div
+                        className="text-[12px] text-gray-500 mt-[2px] m-0"
+                        dangerouslySetInnerHTML={{ __html: item.content }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)